packages/react/README.md

```diff
@@ -29,6 +29,25 @@ const App = () => <h1>Hello World!</h1>;
 export default render(<App />, createContext());
 ```
 
+## `render(reactElement, config)`
```

Comment:
If we consider this the 'consumer' API, i.e. the most approachable way to use `render()`, why do we not mention it first? I really like this API and I think we ought to expose it most prominently.

Reply:
I had the same thoughts, but I had to somehow refer to the original render function for motivation, so it had to come first, I thought. Maybe not a valid point, not sure...